app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/FractionInteractionViewModel.kt

```diff
@@ -33,7 +35,6 @@ class FractionInteractionViewModel private constructor(
   var answerText: CharSequence = ""
```

Comment:
```suggestion
  var answerText: CharSequence = rawUserAnswer?.fraction ?: ""
```

Bit simpler initialization. I suggest also doing this in other interactions where possible.

Reply:
Done